Getting-started.md

```diff
@@ -97,9 +97,6 @@ System.out.println("Number of trades for our strategy: " + tradingRecord.getTrad
 Here is how we can analyze the results of our backtest:
 
 ```java
-// Getting the cash flow of the resulting trades
-CashFlow cashFlow = new CashFlow(series, tradingRecord);
-
```

Comment:
This probably stays in.  I'm guessing the PR is just stale here.

Reply:
Ah yes, I remember.  The unused obj has been removed in `develop` but that change has not yet merged with `master` so the removal probably waits until `master` is updated.  I might suggest a parallel `develop` branch on the wiki to match the `develop` code.  Or it would be really nice if the wiki were somehow integrated with the code in a similar manner to how `CHANGELOG.md` follows `develop`.